### ISSUE TITLE
recommended number of epochs for training BiRefNet HR Matting

### ISSUE THREAD

**Issue by Tilovbek** [ZhengPeng7/BiRefNet#321]
Hi @ZhengPeng7 .
Thank you for sharing such a model.
I would like to ask for your recommendation for my training.
I am training on top of base hr matting model which is **BiRefNet_HR-matting-epoch_135**.
I have trained one model with ~700 images, 100 epochs. the loss dropped from 34.67 to 5.6.
Could you please suggest number of epochs based on the following dataset sizes:
* ~700 images
* ~3000 images 

I also have one more question if possible. Could you tell info about the training dataset size of base model (**BiRefNet_HR-matting-epoch_135**).

Thank you for your time and consideration.
